java-accessapproval/google-cloud-accessapproval-bom/pom.xml

```diff
@@ -9,7 +9,7 @@
   <parent>
     <groupId>com.google.cloud</groupId>
     <artifactId>google-cloud-pom-parent</artifactId>
-    <version>1.1.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-java:current} -->
+    <version>1.1.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-pom-parent:current} -->
```

Comment:
@ddixit14  The script needs to be updated to use google-cloud-java tag.

Reply:
This PR shouldn't update this file. Right?